liquidcore/twofactor/devices.py

```diff
@@ -23,15 +24,17 @@ def delete_all(user, keep=None):
         old_device.delete()
 
 
-def qrencode(data):
-    return subprocess.check_output(['qrencode', data, '-s', '5', '-o', '-'])
-
-
 def qr_png(device, username):
     tpl = 'otpauth://totp/{app}:{username}?secret={secret}&issuer={app}'
     url = tpl.format(
         app=settings.LIQUID_2FA_APP_NAME,
         username=username,
         secret=b32encode(device.bin_key).decode('utf8'),
     )
-    return qrencode(url)
+    img = qrcode.make(url)
+    qr_file = tempfile.NamedTemporaryFile()
+    img.save(qr_file.name, 'PNG')
+    qr_img = qr_file.read()
+    qr_file.close()
```

Comment:
The `qrencode` function was meant to encapsulate the operation "take this url and return a png", even if it was just a one-liner. Now it's more lines so it makes even more sense to encapsulate it. Please bring back the function and move the qrcode logic into it.